packages/salesforcedx-vscode-apex/test/requirements.test.ts

```diff
@@ -8,10 +8,21 @@
 // tslint:disable:no-unused-expression
 
 import { expect } from 'chai';
+import * as path from 'path';
+import * as shell from 'shelljs';
 import { workspace } from 'vscode';
 import { JAVA_HOME_KEY } from '../src/requirements';
 
 describe('Java Requirements Test', () => {
+  it('The jar should be signed', () => {
+    const apexJarPath = path.join(__dirname, '..', 'apex-jorje-lsp.jar');
+    expect(
+      shell
+        .exec(`jarsigner -verify ${apexJarPath}`)
```

Comment:
I need to see if all the CI have this installed. 
If not I will need to think of something else.